src/content/docs/01-concepts/benchmark/index.md

```
@@ -1,91 +1,91 @@
+contributors:
+    - claude
+    - codex
+    - gemini
+---
+
+
+## My Ranking
+
+综合模型能力以及日常使用体验
+
+<img src="https://p.sda1.dev/28/3ef33475959f6e0d989edeecc3c97211/image.png" >
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)